docs/photoniq/vwrs/configure-waiting-rooms.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 20
+title: Configuring Waiting Rooms
+---
+
+This page explains what can be configured in each waiting room using API endpoint properties. For more information about how to create a waiting room, refer to [Configure a Domain](configure-domain.md).
+
+## Entering the Waiting Room
+
+- **Rate Limit:** Set the rate (maximum users per second) at which traffic should be diverted to the waiting room. (`rate_limit` property)
+- **Waiting Room Path:** The URL that the waiting room needs to sit in front of. (`domain_url` property)
+- **Queue Type:** Defines how requests should be removed from the waiting room, such as priority or FIFO (first in, first out). (`queue_type` property)
+
+## In the Waiting Room
+
+- **Waiting Room HTML:** UI that is shown to users in the waiting room. (`waiting_room_path` property)
+- **Waiting Room Position and Time (in HTML):** Include any of the position or time in the waiting room UI. (`????` property)
+- **Waiting Room HTMl Refresh Interval:** How often waiting room UI should be refreshed. (`status_interval` property)
```

Comment:
Maybe add "... refreshed on the user browser"?  Could there be any confusion around where the refresh should take place?

Reply:
Updated